docs/api/data-usage-api.md

```diff
@@ -49,10 +49,10 @@ List([DataUsageBySimCardCollections](/general-information/data-types/#DataUsageB
 
 **Response Payload**
 
-| Field        | Type    | Description                | Required |
-| ------------ | ------- | -------------------------- | -------- |
-| collectionId | UUID    | Unique ID for the SIM card collection we want data usage for | Yes       |
-| year         | Integer | Year to see data usage in | Yes       |
-| month        | String  | Month, in text, to see consumption for                  | Yes       |
+| Field        | Type    | Description                                                  | Required |
+| ------------ | ------- | ------------------------------------------------------------ | ---------|
+| collectionId | UUID    | Unique ID for the SIM card collection we want data usage for | Yes      |
+| year         | Integer | Year to see data usage in                                    | Yes      |
+| month        | String  | Month, in text, to see consumption for                       | Yes      |
 
 [DataUsageBySimCardCollections](/general-information/data-types/#DataUsageBySimCardCollections)
```

Comment:
`#DataUsageBySimCardCollections` must be in lower case. Else the link to section will not work properly.